README.md

```diff
@@ -49,18 +49,18 @@ and then run Docker Compose as follows: ``docker-compose -f docker-compose.yml -
 If you want to enable document sharing via [Etherpad], configure it and run Docker Compose as
 follows: ``docker-compose -f docker-compose.yml -f etherpad.yml up``
 
-If you want to use jibri too, first configure a host as described in JItsi BRoadcasting Infrastructure configuration section
+If you want to use jibri too, first configure a host as described in Jitsi Broadcasting Infrastructure configuration section
```

Comment:
This is intentional I think because it symbolizes Jibri -> JI BR I

Reply:
Thanks, good spot. Undid the change